geomesa-spark/geomesa-spark-geotools/src/test/scala/org/locationtech/geomesa/spark/geotools/GeoToolsSpatialRDDProviderTest.scala

```diff
@@ -43,7 +44,7 @@ class GeoToolsSpatialRDDProviderTest extends Specification {
       val ds = DataStoreFinder.getDataStore(dsParams)
       ingestChicago(ds)
 
-      val rdd = GeoMesaSpark(dsParams).rdd(new Configuration(), sc, dsParams, new Query("chicago"))
+      val rdd = GeoMesaSpark(dsParams).rdd[SimpleFeature](new Configuration(), sc, dsParams, new Query("chicago"))
```

Comment:
maybe just have to add an implicit val in the companion object - not sure how that plays with the implicit defs:
http://stackoverflow.com/questions/12767074/how-to-provide-default-value-for-implicit-parameters-at-class-level